warehouse/dbt_project.yml

```diff
@@ -34,5 +34,14 @@ models:
     # Config indicated by + and applies to all files under models/example/
     +materialized: view
 
-    gtfs_rt:
+    rt_views:
+      schema: views
+
+    gtfs_views_staging:
+      schema: gtfs_views_staging
+
+    gtfs_schedule_latest_only:
+      schema: gtfs_schedule
+
+    gtfs_views:
```

Comment:
Can we alphabetize these model folders? As we get more of them in here, it's going to make the yml easier to scan.

Reply:
Resolved in cc95af9138f52ee6343d6988f456fd5980b17242